frontend/public/components/dashboard/project-dashboard/project-dashboard.tsx

```diff
@@ -9,25 +10,53 @@ import { UtilizationCard } from './utilization-card';
 import { InventoryCard } from './inventory-card';
 import { ActivityCard } from './activity-card';
 import { ProjectDashboardContext } from './project-dashboard-context';
+import { Firehose, FirehoseResult } from '../../utils';
+import { getQuotaResourceTypes, hasComputeResources } from '../../resource-quota';
+import { ResourceQuotaCard } from './resource-quota-card';
 
 const mainCards = [{ Card: StatusCard }, { Card: UtilizationCard }];
 const leftCards = [{ Card: DetailsCard }, { Card: InventoryCard }];
 const rightCards = [{ Card: ActivityCard }];
 
-export const ProjectDashboard: React.FC<ProjectDashboardProps> = ({ obj }) => {
+const ProjectDashboard_: React.FC<ProjectDashboardProps> = ({ obj, resources }) => {
   const context = {
     obj,
   };
 
+  const resourceQuotas = _.get(resources, 'resourceQuotas.data', []);
+  const quotasCards = resourceQuotas
+    .filter((rq) => hasComputeResources(getQuotaResourceTypes(rq)))
+    .map((rq) => ({
+      Card: () => <ResourceQuotaCard resourceQuota={rq} />,
+    }));
+  const cardsWithQuotas = quotasCards.length ? [...mainCards, ...quotasCards] : mainCards;
+
   return (
     <ProjectDashboardContext.Provider value={context}>
       <Dashboard>
-        <DashboardGrid mainCards={mainCards} leftCards={leftCards} rightCards={rightCards} />
+        <DashboardGrid mainCards={cardsWithQuotas} leftCards={leftCards} rightCards={rightCards} />
       </Dashboard>
     </ProjectDashboardContext.Provider>
   );
 };
 
+export const ProjectDashboard = ({ obj }) => {
+  const quotaResources = [
+    {
+      kind: 'ResourceQuota',
+      namespace: obj.metadata.name,
+      isList: true,
+      prop: 'resourceQuotas',
+    },
+  ];
+  return (
+    <Firehose resources={quotaResources}>
+      <ProjectDashboard_ obj={obj} />
+    </Firehose>
+  );
+};
+
 type ProjectDashboardProps = {
   obj: K8sResourceKind;
+  resources?: FirehoseResult;
```

Comment:
I don't think this type is correct. Shouldn't it be something like...?

```
resources: {
  resourceQuotas: FirehoseResult;
};
```

The use of `_.get` above was probably masking this type problem.